bucket/deepstream.json

```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://deepstream.io",
+    "description": "The open realtime server, a fast and secure data-sync realtime server for mobile, web & iot",
+    "license": "AGPL-3.0-or-later",
+    "version": "4.2.5",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/deepstreamIO/deepstream.io/releases/download/v4.2.5/deepstream.io-windows-4.2.5.zip",
+            "hash": "1f5bbfc92abb91795297a0ca7a5cfbf16c18cdfdef518c5265a45aece77de806"
+        }
+    },
+    "bin": "deepstream.exe",
+    "persist": "conf",
+    "env_set": {
```

Comment:
Where is stated you need these environment variables??

Reply:
Here: https://github.com/deepstreamIO/deepstream.io/blob/4dad5d7e2dde6adb1c095faba34506fe6e889961/src/config/js-yaml-loader.ts#L110
https://github.com/deepstreamIO/deepstream.io/blob/4dad5d7e2dde6adb1c095faba34506fe6e889961/src/config/js-yaml-loader.ts#L127

I set the variables to work `deepstream` with a default config and lib folder, instead of change to `%userprofile%\scoop\apps\deepstream\current` and run

Reply:
https://github.com/deepstreamIO/deepstream.io/blob/4dad5d7e2dde6adb1c095faba34506fe6e889961/src/config/js-yaml-loader.ts#L127

```js
(config.libDir && fileUtils.lookupConfRequirePath(config.libDir)) ||
```

You do not have to have libdir environment if you have it inside config. Which is by default set to `../lib`.

![A](https://i.imgur.com/XzGuxIU.png)

Reply:
You do not have to have libdir environment if you have it inside config. Which is by default set to `../lib`.

https://github.com/deepstreamIO/deepstream.io/blob/4dad5d7e2dde6adb1c095faba34506fe6e889961/src/config/js-yaml-loader.ts#L127

```js
(config.libDir && fileUtils.lookupConfRequirePath(config.libDir)) ||
```

![A](https://i.imgur.com/XzGuxIU.png)

Reply:
Nice, I will remove `DEEPSTREAM_LIBRARY_DIRECTORY`